docs/crud.md

```diff
@@ -0,0 +1,103 @@
+# Access a document
+
+#### Request
+```http
+GET /data/:type/:id
+```
+```http
+GET /data/types.cozy.io/events/6494e0ac-dfcb-11e5-88c1-472e84a9cbee
```

Comment:
Why setting `cozy.io` in the type name ? eg. `/data/types.cozy.io/events/123`
Shouldn't it be simpler just named `types` ? eg. `/data/types/events/123`
One reason I see is having many kind of types and not all managed by Cozy, which is also wrong because the value can be rewritten by the couchDb access layer.